packages/nimble-components/src/patterns/button/styles.ts

```diff
@@ -63,9 +63,22 @@ export const styles = css`
         outline: none;
         margin: 0;
         padding: 0 ${standardPadding};
+        position: relative;
         transition: box-shadow ${smallDelay};
```

Comment:
I added a `smallDelay` transition to the outer border, a `mediumDelay` to the box-shadow, and a 0.5s transition to the transition for the outline used in the focus visible state. These changes should work for every button including the Anchor and Toggle buttons. 

The outline in the focus visible state is somewhat flickery, but I initially ran into issues adding a transition because it would create a black outline until the transition animation finished, and the only way to fix this was to make the outline transparent with the `color` css variable, which when used in the same class as the text would cause the text to be transparent as well. I found a fix for this, and as I show in the video it's not very noticeable when you're zoomed out.

https://github.com/ni/nimble/assets/103057880/32a6fe51-211f-4a51-a8b2-978ed02ab685